packages/ui-library/src/theme/typography.ts

```diff
@@ -125,7 +125,7 @@ export const buildTypographyVariant = (mode: PaletteMode): TypographyOptions =>
     fontFamily: 'Inter',
     fontSize: '0.875rem', // 14/16
     fontStyle: 'normal',
-    fontWeight: 400,
+    fontWeight: 500,
```

Comment:
hold up there cowboy, this will surely break other styles

Reply:
Haha 😄 . I checked this in detail, I truly think this changes (all 3 font weight related) are aligned with the design

Reply:
Apart from that, what other part of the app that used this is now not according to the design?